docs/en/operations/table_engines/custom_partitioning_key.md

```diff
@@ -1,46 +1,36 @@
-
 # Custom Partitioning Key
 
-The partition key can be an expression from the table columns, or a tuple of such expressions (similar to the primary key). The partition key can be omitted. When creating a table, specify the partition key in the ENGINE description with the new syntax:
+Partitioning is available for the [MergeTree](mergetree.md) family tables (including [replicated](replication.md) tables). [Materialized views](materializedview.md) based on MergeTree table support partitioning as well.
 
-```
-ENGINE [=] Name(...) [PARTITION BY expr] [ORDER BY expr] [SAMPLE BY expr] [SETTINGS name=value, ...]
-```
+A partition is a logic combination of records in a table by a specified criterion. You can set a partition by an arbitrary criterion. For example: by month, by day, by event type and so on. Each partition is stored separately in order to simplify manipulations with this data. When accessing the data ClickHouse only uses the corresponding partition.
```

Comment:
* logic -> logical?
* Second sentence does not give any new information.
* the corresponding partition -> as small subset of partitions as possible.